runtime/doc/dev_vim_patches.txt

```diff
@@ -0,0 +1,303 @@
+*dev-vim-patches.txt*                            Merging patches from upstream
+
+                            NVIM REFERENCE MANUAL
+
+See |dev-vim-patches-quickstart| to get started immediately. Also note the
+section on Not Applicable patches.
+
+Nvim was forked from Vim 7.4.160; it is kept up-to-date with relevant Vim
+patches in order to avoid duplicate work. Run `vim-patch.sh`
+https://github.com/neovim/neovim/blob/master/scripts/vim-patch.sh to see the
+status of Vim patches:
+
+>bash
+    ./scripts/vim-patch.sh -l
+<
+
+Everyone is welcome to |dev-vim-patches-pull-requests| for relevant Vim
+patches, but some types of patches are |dev-vim-patches-not-applicable|.
+
+==============================================================================
+QUICKSTART                                        *dev-vim-patches-quickstart*
+
+1. Pull down the Nvim source:
+>bash
+    git clone https://github.com/neovim/neovim.git
+<
+2. Run `./scripts/vim-patch.sh -l` to see the list of missing Vim patches.
+
+3. Choose a patch from the list (usually the _oldest_ one), e.g. `8.0.0123`.
+
+- Check for open vim-patch PRs
+  https://github.com/neovim/neovim/pulls?q=is%3Apr+is%3Aopen+label%3Avim-patch.
+
+4. Run `./scripts/vim-patch.sh -p 8.0.0123`
+
+5. Follow the instructions given by the script.
+
+NOTES ~
+
+- It's strongly recommended to work on the _oldest_ missing patch, because
+  later patches might depend on the changes.
+- Use `git log --grep` or `git log -G` to search the Nvim/Vim source history
+  (even _deleted_ code). E.g. to find `reset_option_was_set`:
+
+>bash
+    git log -p -G reset_option_was_set
+<
+- Pass `git log` options like `--grep` and `-G` to `./scripts/vim-patch.sh -L`
+  to filter unmerged Vim patches E.g. to find `+quickfix` patches:
+
+>bash
+    ./scripts/vim-patch.sh -L --grep quickfix -- src
+<
+==============================================================================
+PULL REQUESTS                                  *dev-vim-patches-pull-requests*
+
+Note: vim-patch.sh automates these steps for you. Use it!
+
+- Install gh https://cli.github.com/ or git-hub if you want to use
```

Comment:
```suggestion
- Install gh (https://cli.github.com/) or git-hub if you want to use
```

Reply:
I think `gh` won that race? (These PRs are an excellent opportunity for pruning outdated or obsolete information.)

Reply:
Eh, sure. We can remove git-hub

Reply:
Although I still think you can use git-hub if you want to so maybe it doesn't make sense to remove it.

Reply:
Sure. I do think it makes sense to suggest `gh` more strongly, as by now it works quite well and does make opening PRs much smoother (especially if you can remember the `-B` option...)

Reply:
OK. Removed git-hub mention